packages/sdk-utils/test/helpers.js

```diff
@@ -26,6 +26,12 @@ const helpers = {
     return Array.isArray(res.body[what]) ? res.body[what].map(map) : map(res.body);
   },
 
+  async mockGetCurrentUrl() {
+    try {
+      await utils.request('/wd/hub/session');
+    } catch {}
+  },
+
```

Comment:
Where is this used?

Reply:
This is used in test case by js SDK.https://github.com/percy/percy-selenium-js/pull/333/files#diff-8a8c49bdbe55241012d9ad47a61d9f11f11a2be0105f967811a8f2361f088b08R84-R85